azurerm/internal/services/containers/container_group_resource_test.go

```diff
@@ -872,6 +877,15 @@ resource "azurerm_container_group" "test" {
   ip_address_type     = "public"
   os_type             = "Linux"
 
+  exposed_port {
+    port = 80
+  }
+
+  exposed_port {
+    port     = 5443
+    protocol = "UDP"
+  }
+
```

Comment:
as above, can we split this out into an explicit update test for this property?

Reply:
Done.